try {
    fuck();
} catch (e) {
    console.log(e);
} finally {
    console.log("Finally");
}
